setup.py

```diff
@@ -71,7 +71,6 @@ def get_install_requires():
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Topic :: System :: Networking',
-        'Programming Language :: Python :: 2.7',
```

Comment:
look also for `sys.version_info.major` in this file

Reply:
Review Please :smile: